py/desispec/scripts/submit_night.py

```diff
@@ -444,3 +461,46 @@ def submit_night(night, proc_obstypes=None, z_submit_types=None, queue='realtime
             write_table(ptable, tablename=proc_table_pathname)
 
     print(f"Completed submission of exposures for night {night}.", '\n\n\n')
+
+
+def get_completed_tiles(specstatus_path=None):
```

Comment:
Please add a unit test for this function, that would write a temporary specstatus file covering all of the different cases and then calls this function with that file and confirms that it produces the expected answer.

Reply:
Resolved in 982afa5